src/index.js

```diff
@@ -1,19 +1,20 @@
 import express from 'express';
-import { serve, setup } from 'swagger-ui-express';
-import swaggerJsdoc from 'swagger-jsdoc';
-import swaggerDefinition from './swagger';
+import bodyParser from 'body-parser';
+import registerApiDocs from './swagger';
+import router from './routes/auth';
 
 const app = express();
 
-const swaggerOptions = {
-  customSiteTitle: 'The Spinners',
-  explorer: true,
-};
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
 
-const swaggerSpecs = swaggerJsdoc(swaggerDefinition);
-app.use('/docs', serve, setup(swaggerSpecs, swaggerOptions));
+app.use(router);
+
+registerApiDocs(app);
 
 const PORT = process.env.PORT || 3000;
 
 // eslint-disable-next-line no-console
 app.listen(PORT, () => console.log(`Listening on port ${PORT}.......`));
+
+export default app;
```

Comment:
Is there a reason why you are exporting `app`? If yes, you shouldn't start the app here. Instead, create a `server.js` file where you can start the server.

The implication of this is that the app will be started and the port will be unavailable if you use it with supertest or chai-http.

Reply:
@higustave-ops ☝️

Reply:
I think the reason we were exporting the app is to access it from routes.

Reply:
The message is not visible to gustave

Reply:
Move line 17 to another file called `server.js` instead.